apps/right-to-rent-check/behaviours/get-declarer.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    if (req.sessionModel.get('representative') === 'landlord') {
+      locals.declarer = req.translate('pages.declaration.landlord');
+      locals.declarerID = 'landlord';
```

Comment:
use camelcase

Reply:
Do you mean `locals.declareId` rather than `declareID`?